exams/pearson.py

```diff
@@ -36,6 +41,13 @@
 log = logging.getLogger(__name__)
 
 
+def _format_date(date):
```

Comment:
why `_format_date`, `_format_datetime` and `_get_field_mapper` are not part of `TSVWriter`? Are they used outside this module?